package.json

```
@@ -43,6 +43,6 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
-    "node-abort-controller": "^1.2.1"
+    "node-abort-controller": "^2.0.0"
```

It would be nice to accept both major versions:

```suggestion
    "node-abort-controller": "^1.2.1 || ^2.0.0"
```